core/trino-main/src/main/java/io/trino/metadata/Metadata.java

```diff
@@ -233,12 +233,12 @@ Optional<TableExecuteHandle> getTableHandleForExecute(
     /**
      * Rename the specified column.
      */
-    void renameColumn(Session session, TableHandle tableHandle, ColumnHandle source, String target);
+    void renameColumn(Session session, TableHandle tableHandle, QualifiedObjectName table, ColumnHandle source, String target);
```

Comment:
Why do we take `QualifiedObjectName` instead of `CatalogSchemaTableName`? The subsequent logic uses only `CatalogSchemaTableName`.

Reply:
I was attempting to keep consistency with what I saw elsewhere (such as [renameTable](https://github.com/starburstdata/galaxy-trino/blob/galaxy/core/trino-main/src/main/java/io/trino/metadata/Metadata.java#L205)), but looking elsewhere, you're right. I'll fix.